GitUI/SpellChecker/EditNetSpell.cs

```diff
@@ -870,16 +870,32 @@ private string GetWordAtCursor()
             if (TextBox.Text.Length > TextBox.SelectionStart && IsSeparatorChar(TextBox.Text[TextBox.SelectionStart]))
                 return null;
 
+            return ExtractWord(TextBox.Text, TextBox.SelectionStart - 1);
+        }
+
+        private string ExtractWord(string text, int cursor)
+        {
```

Comment:
check for cursor < 0 first and return `string.Empty`. this will save on unnecessary allocation of `StringBuilder`